src/representation_theory/group_cohomology_resolution.lean

```diff
@@ -216,10 +223,130 @@ def of_mul_action_basis  :
+    comm' := λ g, by ext; convert congr_fun ((s.π.app (some j)).comm' g) i },
+  fac' := λ s j, option.cases_on j (by ext) (λ j, by ext; refl),
+  uniq' := λ s f h, by ext x j; simpa only [←h (some j)] }
+
+instance has_wide_pullback : has_wide_pullback (arrow G).right
+  (λ i : fin (m.unop.len + 1), (arrow G).left) (λ i, (arrow G).hom) :=
+⟨⟨⟨wide_cospan.cone G m, wide_cospan.is_limit G m⟩⟩⟩
+
+/-- The `m`th object of the Čech nerve of the `G`-set hom `G ⟶ {pt}` is isomorphic to `Gᵐ⁺¹` with
+the diagonal action. -/
+def cech_nerve_obj_iso :
+  (cech_nerve (arrow G)).obj m ≅ Action.of_mul_action G (fin (m.unop.len + 1) → G) :=
+is_limit.cone_point_unique_up_to_iso (limit.is_limit _) (wide_cospan.is_limit G m)
+
+/-- The Čech nerve of the `G`-set hom `G ⟶ {pt}` is naturally isomorphic to `EG`, the universal
+cover of the classifying space of `G` as a simplicial `G`-set. -/
+def cech_nerve_iso : cech_nerve (arrow G) ≅ classifying_space_universal_cover G :=
+(@nat_iso.of_components _ _ _ _ (classifying_space_universal_cover G) (cech_nerve (arrow G))
+(λ n, (cech_nerve_obj_iso G n).symm) $ λ m n f,  wide_pullback.hom_ext (λ j, (arrow G).hom) _ _
+(λ j, begin
+  simp only [category.assoc],
+  dsimp,
+  rw wide_pullback.lift_π,
+  erw [limit.cone_point_unique_up_to_iso_hom_comp (wide_cospan.is_limit G n),
+    limit.cone_point_unique_up_to_iso_hom_comp (wide_cospan.is_limit G m)],
+  refl,
+end) (by ext)).symm
+
+end classifying_space_universal_cover
+namespace group_cohomology.resolution
+
+section differential
+variables (k G)
+/-- The `k`-linear map underlying the differential in the standard resolution of `k` as a trivial
+`k`-linear `G`-representation. It sends `(g₀, ..., gₙ) ↦ ∑ (-1)ⁱ • (g₀, ..., ĝᵢ, ..., gₙ)`. -/
+def d (n : ℕ) : ((fin (n + 1) → G) →₀ k) →ₗ[k] ((fin n → G) →₀ k) :=
+finsupp.lift ((fin n → G) →₀ k) k (fin (n + 1) → G) (λ g, (@finset.univ (fin (n + 1)) _).sum
+  (λ p, finsupp.single (g ∘ p.succ_above) ((-1 : k) ^ (p : ℕ))))
+
+variables {k G}
+
+@[simp] lemma d_of {n : ℕ} (c : fin (n + 1) → G) :
+  d k G n (finsupp.single c 1) = finset.univ.sum (λ p : fin (n + 1), finsupp.single
+    (c ∘ p.succ_above) ((-1 : k) ^ (p : ℕ))) :=
+by simp [d]
+
+end differential
+end group_cohomology.resolution
+variables (k G) [monoid G]
+
+/-- The standard resolution of `k` as a trivial representation, defined as the alternating
+face map complex of a simplicial `k`-linear `G`-representation. -/
+def group_cohomology.resolution := (algebraic_topology.alternating_face_map_complex (Rep k G)).obj
+  (classifying_space_universal_cover G ⋙ (Rep.linearization k G).1.1)
+
+namespace group_cohomology.resolution
+
+/- Leaving this here for now - not sure it should exist or where it should go. Everything I tried
+to avoid this lemma was messy or gave me weird errors. -/
+lemma int_cast_smul {k V : Type*} [comm_ring k] [add_comm_group V] [module k V] (r : ℤ) (x : V) :
+  (r : k) • x = r • x :=
+algebra_map_smul k r x
```

Comment:
I do not see how to avoid this lemma. Presumably, it could be moved to `algebra/algebra/basic.lean`.